CHANGELOG.md

```diff
@@ -4,6 +4,105 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [v18.5.0] 2024-09-25
```

Comment:
@npauzenga do we actually want the v19.0.0 release in this CL at all since it's on the v18.5.x branch?

Reply:
Good question. No 😄 

I don't think it super matters but that's likely why I'm not used to seeing the backport version above the latest forward version, because the latest forward isn't usually here.